src/main/java/gui/GUIDriver.java

```diff
@@ -3,10 +3,7 @@
 import com.sun.source.tree.BreakTree;
 import gui.utility.InstanceMapper;
 import gui.utility.NavigationHelper;
-import gui.view.AddTaskPageController;
-import gui.view.MonthlyCalendarController;
-import gui.view.TodoListPageController;
-import gui.view.WeeklyCalendarController;
+import gui.view.*;
```

Comment:
Avoid wild card import

Reply:
You could change your IDE setting so it doesn't automatically merge imports into a wild card import

Reply:
edited!